models/classes/Lists/Business/Domain/ValueCollectionSearchRequest.php

```diff
@@ -147,6 +150,23 @@ public function addExcluded(string $excluded): self
         return $this;
     }
 
+    public function hasOffset(): bool
+    {
+        return $this->offset > 0;
```

Comment:
0 is a valid offset, I did not get your validation. Suggestion: 

```suggestion
        return $this->offset >= 0;
```

An you can make the default null

Reply:
Thanks, added. A bit modified your solution by additional check for null:
```php
return $this->offset !== null && $this->offset >= 0;
```
Because `null >= 0` is `true` - just to avoid this 🙂